test/VarianceTest.cc

```diff
@@ -341,19 +342,141 @@ INSTANTIATE_TEST_CASE_P(
+                      &eb_aom_variance16x16_avx2),
+        VarianceParam(16, 32, &eb_aom_variance16x32_c,
+                      &eb_aom_variance16x32_avx2),
+        VarianceParam(16, 64, &eb_aom_variance16x64_c,
+                      &eb_aom_variance16x64_avx2),
         VarianceParam(32, 8, &eb_aom_variance32x8_c, &eb_aom_variance32x8_avx2),
-        VarianceParam(32, 16, &eb_aom_variance32x16_c, &eb_aom_variance32x16_avx2),
-        VarianceParam(32, 32, &eb_aom_variance32x32_c, &eb_aom_variance32x32_avx2),
-        VarianceParam(32, 64, &eb_aom_variance32x64_c, &eb_aom_variance32x64_avx2),
-        VarianceParam(64, 16, &eb_aom_variance64x16_c, &eb_aom_variance64x16_avx2),
-        VarianceParam(64, 32, &eb_aom_variance64x32_c, &eb_aom_variance64x32_avx2),
-        VarianceParam(64, 64, &eb_aom_variance64x64_c, &eb_aom_variance64x64_avx2),
-        VarianceParam(64, 128, &eb_aom_variance64x128_c, &eb_aom_variance64x128_avx2),
-        VarianceParam(128, 64, &eb_aom_variance128x64_c, &eb_aom_variance128x64_avx2),
+        VarianceParam(32, 16, &eb_aom_variance32x16_c,
+                      &eb_aom_variance32x16_avx2),
+        VarianceParam(32, 32, &eb_aom_variance32x32_c,
+                      &eb_aom_variance32x32_avx2),
+        VarianceParam(32, 64, &eb_aom_variance32x64_c,
+                      &eb_aom_variance32x64_avx2),
+        VarianceParam(64, 16, &eb_aom_variance64x16_c,
+                      &eb_aom_variance64x16_avx2),
+        VarianceParam(64, 32, &eb_aom_variance64x32_c,
+                      &eb_aom_variance64x32_avx2),
+        VarianceParam(64, 64, &eb_aom_variance64x64_c,
+                      &eb_aom_variance64x64_avx2),
+        VarianceParam(64, 128, &eb_aom_variance64x128_c,
+                      &eb_aom_variance64x128_avx2),
+        VarianceParam(128, 64, &eb_aom_variance128x64_c,
+                      &eb_aom_variance128x64_avx2),
         VarianceParam(128, 128, &eb_aom_variance128x128_c,
                       &eb_aom_variance128x128_avx2)));
 
-}  // namespace
+typedef std::tuple<int, int> BlockSize;
+BlockSize TEST_BLOCK_SIZES[] = {
+    BlockSize(4, 4),    BlockSize(4, 8),    BlockSize(4, 16),
+    BlockSize(8, 4),    BlockSize(8, 8),    BlockSize(8, 16),
+    BlockSize(8, 32),   BlockSize(16, 4),   BlockSize(16, 8),
+    BlockSize(16, 16),  BlockSize(16, 32),  BlockSize(16, 64),
+    BlockSize(32, 8),   BlockSize(32, 16),  BlockSize(32, 32),
+    BlockSize(32, 64),  BlockSize(64, 16),  BlockSize(64, 32),
+    BlockSize(64, 64),  BlockSize(64, 128), BlockSize(128, 64),
+    BlockSize(128, 128)};
+/**
+ * @brief Unit test for variance functions, target functions include:
+ *  - highbd_variance64_{c,avx2}
+ *
+ * Test strategy:
+ *  This test case contains zero test, random value test, one quarter test as
+ * test pattern.
+ *
+ *
+ * Expect result:
+ *  Results come from reference functon and target function are
+ * equal.
+ *
+ * Test coverage:
+ *
```

Comment:
Maybe we should add different block size as coverage;

Reply:
done.